spec/index.html

```diff
@@ -295,13 +315,26 @@ <h3>Terms defined by this specification</h3>
       <dt><dfn data-lt="input dataset|input datasets">input dataset</dfn></dt>
       <dd>The abstract <a>RDF dataset</a> that is provided as input to
         the algorithm.</dd>
+      <dt><dfn>input blank node identifier map</dfn></dt>
+      <dd>Records any blank node identifiers already assigned to the
+        <a>input dataset</a>.
+        If the <a>input dataset</a> is provided as an N-Quads document,
+        the <a>map</a> relates blank nodes in the abstract <a>input dataset</a>
+        to the blank node identiers used within the N-Quads document,
+        otherwise, identifiers are assigned arbitrarily for
+        each blank node in the input dataset not previously identified.
+        <div class="note">Some implementations may retain blank node identifiers
+          in the parsed dataset.</div>
```

Comment:
```suggestion
        <div class="note">Some implementations may retain blank node identifiers
          of the parsed dataset.</div>
```

Reply:
Is "might concretely deal with" correct in English? It sounds a bit strange to my ears. Can't we just say "might deal with" ?

Reply:
Yeah, it's fine to simplify that -- I removed `concretely`.